source/margarita/upgrading.rst

```diff
@@ -0,0 +1,467 @@
+
+   .. code-block:: diff
+
+      Modified   manage.py
+      diff --git a/manage.py b/manage.py
+      index cb48c9e..8bc2fce 100644
+      --- a/manage.py
+      +++ b/manage.py
+      @@ -4,2 +4,5 @@ import sys
+
+      +import dotenv
+      +dotenv.read_dotenv()
+      +
+       if __name__ == "__main__":
+
+#. Modify ``requirements/base.txt``:
+
+   .. code-block:: diff
+
+      Modified   requirements/base.txt
+      diff --git a/requirements/base.txt b/requirements/base.txt
+      index 2bb6ff2..ca74917 100644
+      --- a/requirements/base.txt
+      +++ b/requirements/base.txt
+      +
+      +django-dotenv==1.3.0
+
+#. Modify ``README.rst`` (and follow those instructions for your local setup):
+
+   .. code-block:: diff
+
+      Modified   README.rst
+      diff --git a/README.rst b/README.rst
+      index c45b564..8c456be 100644
+      --- a/README.rst
+      +++ b/README.rst
+      @@ -27,7 +27,9 @@ necessary requirements::
+
+      -Then create a local settings file and set your ``DJANGO_SETTINGS_MODULE`` to use it::
+      +Next, we'll set up our local environment variables. We use `django-dotenv
+      +<https://github.com/jpadilla/django-dotenv>`_ to help with this. It reads environment variables
+      +located in a file name ``.env`` in the top level directory of the project. The only variable we need
+      +to start is ``DJANGO_SETTINGS_MODULE``::
+
+      -    cp myproject/settings/local.example.py myproject/settings/local_dev.py
+      -    echo "export DJANGO_SETTINGS_MODULE=myproject.settings.local_dev" >> $VIRTUAL_ENV/bin/postactivate
+      -    echo "unset DJANGO_SETTINGS_MODULE" >> $VIRTUAL_ENV/bin/postdeactivate
+      +    (myproject)$ cp myproject/settings/local.example.py myproject/settings/local.py
+      +    (myproject)$ echo "DJANGO_SETTINGS_MODULE=myproject.settings.local" > .env
+
+
+Update ALLOWED_HOSTS
+--------------------
+
+Find the ``ALLOWED_HOSTS`` setting (probably in ``staging.py``) and change it to use ``DOMAIN``:
+
+.. code-block:: diff
+
+   Modified   myproject/settings/staging.py
+   diff --git a/myproject/settings/staging.py b/myproject/settings/staging.py
+   index db7b3b4..be4024d 100644
+   --- a/myproject/settings/staging.py
+   +++ b/myproject/settings/staging.py
+   @@ -34,3 +34,3 @@ SESSION_COOKIE_HTTPONLY = True
+
+   -ALLOWED_HOSTS = os.environ.get('ALLOWED_HOSTS', '').split(';')
+   +ALLOWED_HOSTS = [os.environ['DOMAIN']]
+
+Frontend Improvments
+--------------------
+
+Prepare for Calvin's frontend improvements. Add a *dummy* ``package.json`` which can be updated
+later. Until it is updated, the frontend improvements won't take effect:
+
+.. code-block:: json
+
+   {
+     "name": "",
+     "version": "0.0.0",
+     "description": "",
+     "main": "",
+     "engines" : {
+       "node" : ">=4.2 <4.3"
+     },
+     "scripts": {
+       "build": "true"
+     },
+     "author": "",
+     "license": "",
+     "dependencies": {},
+     "devDependencies": {}
+   }
+
+Deploy!!!
```

Comment:
Need to commit the changes to some branch, push it to github, then update staging.sls to deploy that branch temporarily.